src/c/BuscaSequencial.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+int buscaSequencial(char vetor[], int tamanho, char buscado  ) {
+    int i;
+    for (i = 0; i < tamanho; i++)
```

Comment:
Uma boa prática é declarar o `i` no próprio `for` para limitar o escopo:
```suggestion
    for (int i = 0; i < tamanho; i++)
```